tensorflow/compiler/xla/service/cpu/runtime_conv2d_mkl.cc

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 #ifdef ENABLE_MKL
 #include <omp.h>
 
-#include "mkldnn.hpp"
+#include "dnnl.hpp"
```

Comment:
Having some difficulty in using full path. I tried the following options:
       (1) third_party/mkl_dnn/include/oneapi/dnnl/dnnl.hpp   or
       (2) third_party/mkl_dnn_v1/include/oneapi/dnnl/dnnl.hpp or 
       (3) third_party/mkl_dnn/include/dnnl.hpp   or
       (4) third_party/mkl_dnn_v1/include/dnnl.hpp
Options 3 & 4 should be same as 1 & 2.  Somehow bazel could not locate the header file.

I guess that we may have to change some bazel configuration files (have tried "absl" way for Eigen but no success :))

So I will continue to work on this suggestion.

Reply:
@penpornk   Both of the following options (without third_party/<prod name> work
  Option 1:   #include "include/dnnl.hpp"      
  Option 2:   #include "include/oneapi/dnnl/dnnl.hpp"
 Here include/dnnl.hpp indirectly points to nclude/oneapi/dnnl/dnnl.hpp.

 Option 2 is a better choice as it is more specific. However, the concern is that oneDNN may change the path in the future  ("oneapi is too generic; while "dnnl" has already replaced by "oneDNN" as product name). 

 My suggestion is to keep the old way without adding path. What do you think?

Reply:
Oh, sorry I didn't think it would be this inconvenient. Let's keep it like this for now then. 
I think there is an easier way because Eigen could use the full path. I'll look into it after this PR. :)